homeassistant/components/fibaro/cover.py

```diff
@@ -37,9 +37,9 @@ def bound(position):
         if position is None:
             return None
         position = int(position)
-        if position <= 5:
+        if position <= 1:
```

Comment:
@baskalex do you have an idea why the margin was set to 5 % to begin with?

Reply:
Nope, absolutely no idea. But its a very strange solution if you ask me :)

Reply:
@pbalogh77 do you know why the margin is 5 %.